test/handler/test_vulture_handler.vader

```diff
@@ -0,0 +1,28 @@
+Before:
+  runtime ale_linters/python/vulture.vim
+
+After:
+  Restore
+
+  call ale#linter#Reset()
+
+  silent file something_else.py
+
+Execute(Basic vulture check should be handled):
+  call ale#test#SetFilename('something_else.py')
+  AssertEqual
+  \ [
+  \   {
+  \     'lnum': 34,
+  \     'text': "unused variable 'foo' (60% confidence)",
+  \     'type': 'W',
+  \   },
+  \ ],
+  \ ale_linters#python#vulture#Handle(bufnr(''), [
+  \   "/testplugin/test/handler/something_else.py:34: unused variable 'foo' (60% confidence)",
```

Comment:
@w0rp is there any helper function to get that path dynamically in tests instead of hardcoding it?

it seems to be the reason of appveyor testsuite to fail

Reply:
Look at existing tests for uses of `SetDirectory`, `RestoreDirectory`, and `SetFilename`. They will set up paths so an filenames so they work inside and outside of the Docker app. Use the `g:dir` variable and `ale#path#Simplify` so the paths work on both Linux and Windows.